documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -104,6 +104,14 @@ Details are in the following topics:
 
 endif::product[]
 
+// Type: concept
+// ModuleID: creating-debezium-postgresql-user
+// Title: Security for PostgreSQL connector
+[[postgresql-security]]
+=== Security
+
+See link:https://www.postgresql.org/docs/current/logical-replication-security.html for details, but in general the debezium user needs to have `replication` privileges. It is possible to just provide the user with `superuser` privileges but that allows the user complete control of the PostgreSQL instance. Instead create the the debezium user with `replication` privileges and add SELECT privileges on the table(s) as well as CREATE privileges on the database.
```

Comment:
```suggestion
See link:https://www.postgresql.org/docs/current/logical-replication-security.html for details, but in general the {prodname} user needs to have `replication` privileges. It is possible to just provide the user with `superuser` privileges but that allows the user complete control of the PostgreSQL instance. Instead create the {prodname} user with `replication` privileges and add SELECT privileges on the table(s) as well as CREATE privileges on the database.
```

Reply:
Could you also add a descriptive label to the link, so it doesn't get rendered as the plain URL.